nilearn/masking.py

```diff
@@ -522,8 +522,10 @@ def compute_multi_background_mask(data_imgs, border_size=2, upper_cutoff=0.85,
 
 
 @deprecated("Function 'compute_gray_matter_mask' has been renamed to "
-            "'compute_brain_mask' and "
-            "'compute_gray_matter_mask' will be removed in release 0.9.")
+            "'compute_brain_mask' and 'compute_gray_matter_mask' will be "
+            "removed in release 0.9.0. The default resolution of the "
+            "estimated mask will also change in release 0.9.0 from 2mm to "
+            "1mm.")
```

Comment:
I don't think we need to change this message. The resolution warnings would be handled by the template loading functions which are called from this one.